cypress/integration/specialFeatures/scriptSwitching/index.js

```diff
@@ -14,7 +14,9 @@ Object.keys(config)
   .forEach(serviceId => {
     const { variant } = config[serviceId];
     const serviceName = config[serviceId].name;
-    const otherVariant = appConfig[serviceName][variant].scriptLink.variant;
+    const otherVariant = appConfig[serviceName][variant];
+    const scriptLinkVariant =
+      otherVariant.scriptLink && otherVariant.scriptLink.variant;
```

Comment:
I think this is clearer to read:
```suggestion
    const scriptLink = appConfig[serviceName][variant].scriptLink;
    const scriptLinkVariant = scriptLink && scriptLink.variant;
```